src/main/java/org/killbill/billing/plugin/vertex/VertexTaxCalculator.java

```diff
@@ -307,7 +319,7 @@ private SaleRequestType toTaxRequest(final Account account,
                                          final Iterable<PluginProperty> pluginProperties,
                                          final LocalDate taxItemsDate,
                                          final String companyName,
-                                         final String companyDivision,
+                                         final String defaultCompanyDivision,
```

Comment:
Also we don't need company name and division to be flowing from `VertexApiClient`  class as we have properties now

Reply:
we should simplify the `VertexApiClient` class by removing these properties.

Reply:
I agree to remove division as this is dynamic field, but as companyName is common I think we should keep it